tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Differ;
+
+use PhpCsFixer\Differ\SebastianBergmannDiffer;
+use PHPUnit\Framework;
+use Prophecy\Argument;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Differ\SebastianBergmannDiffer
+ */
+final class SebastianBergmannDifferTest extends Framework\TestCase
+{
+    public function testIsDiffer()
+    {
+        $differ = new SebastianBergmannDiffer();
+
+        $this->assertInstanceOf('PhpCsFixer\Differ\DifferInterface', $differ);
+    }
+
+    public function testDiffReturnsDiff()
```

Comment:
actually, exactly the same for next test class, too much of testing the internal implementation details instead of public interface